File: references/admin_templates/template_1/CakePHP/CakePHP/src/Controller/AuthenticationController.php
<?php
declare(strict_types=1);

namespace App\Controller;

/**
 * Authentication Controller
 *
 */
class AuthenticationController extends AppController
{
    public function forgotPassword()
    {
        $this->viewBuilder()->setLayout('layout2');
    }

    public function signin()
    {
        $this->viewBuilder()->setLayout('layout2');
    }

    public function signup()
    {
        $this->viewBuilder()->setLayout('layout2');
    }
}

File: references/admin_templates/template_1/MVC/MVC/Controllers/AuthenticationController.cs
using Microsoft.AspNetCore.Mvc;

namespace WowDash.Controllers
{
    public class AuthenticationController : Controller
    {
        public IActionResult ForgotPassword()
        {
            return View();
        }
        public IActionResult Signin()
        {
            return View();
        }
        public IActionResult Signup()
        {
            return View();
        }
    }
}
